service/internal/resources.go

```diff
@@ -0,0 +1,323 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by "esc -pkg internal -o resources.go templates/"; DO NOT EDIT.
```

Comment:
I can avoid submitting this generated code. Add it as required step in make to re-generate this file if not present, and add it to .gitignore.

Reply:
Yes, also I could not find a way to add the copyright so an extra step is required. I kind of like it to generate this every time, but that may be annoying for ide users if they don't use make

Reply:
What about a test to ensure that the version matches all the template files (just compare FS(true) vs FS(false))? In case of failure, the test should tell how to re-generate the file (including make to add header).

Reply:
Was not trivial calling twice to create a template from the same data returns different objects that the assert.Equal sees them as different, sorry I will merge as is, we can improve later.